src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/ILoggerWithColor.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using NuGet.Common;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal interface ILoggerWithColor : ILogger
+    {
+        void LogMinimalWithColor(string data, ConsoleColor color);
```

Comment:
Since the type implies it supports color, you could just overload the method names like this:
```suggestion
        void LogMinimal(string data, ConsoleColor color);
```
![image](https://github.com/NuGet/NuGet.Client/assets/49205731/d6e5c43d-50df-42f5-abf9-0774d0bcd685)

Reply:
That is a good point. But LogMinimalWithColor does more than adding color. It uses Console.Write rather than Console.WriteLine. If that would not cause confusion, I can overload the LogMinimal method